packages/deployer/test/sample-project/contracts/mixins/ModulesMixin.sol

```diff
@@ -0,0 +1,17 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../storage/ProxyStorage.sol";
+import "../interfaces/IRouter.sol";
+
+contract ModulesMixin is ProxyStorageNamespace {
```

Comment:
This is an experimental type 3 inter module implementation that we decided to drop. We should do research on this, but its not part of 3.0.0